src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyCommon/SwarmInputNodes.py

```diff
@@ -186,25 +192,82 @@ def INPUT_TYPES(s):
     def do_input(self, value, **kwargs):
         return (value, )
 
+def load_image(image):
```

Comment:
don't need the entire loadimage func here, just _call_ the standard func from inside comfy source (don't subclass just call the func)